docs/docs/dev-guide-new-shield.md

```diff
@@ -64,6 +64,8 @@ endif
 
 ## Shield Overlay
 
+![Labelled Pro Micro pins](pro-micro-labelled-image.jpg)
```

Comment:
In addition to the image itself, we should include some description of how the DT nodes map to the labelled pins in the image., Perhaps something along the lines of:

"ZMK uses the green Arduino pin names ,e.g. `D0`, or `A1`, as the reference to generate references to specific pro-micro pin locations. For example, the location labeled `D0` in this diagram can be referred to in the ZMK devicetree files as `&pro_micro_d 0`."

Thoughts?

Reply:
I could add that to the documentation. I will write some improvements and post it here. Though now i have a fair idea of what is happening thanks to you.

Reply:
"ZMK uses the green color coded pin names to generate devicetree node references. For example, to refer to the node `D0` in the devicetree files, use `&pro_micro_d 0`."
Is this okay?

Reply:
Can we give a second example for an A pin as well? Want to be sure we cover that as well. Thanks!

Reply:
"ZMK uses the green color coded pin names to generate devicetree node references. For example, to refer to the node `D0` in the devicetree files, use `&pro_micro_d 0` or to refer to `A1`, use `&pro_micro_a 1`."
Does this work?